src/simple_continuous_time_system.py

```diff
@@ -1,3 +1,49 @@
+#!/usr/bin/env python
+
+import numpy as np
 import pydrake
+from pydrake.systems.framework import (DiagramBuilder, VectorSystem)
```

Comment:
BTW Would you want to use something like https://github.com/RobotLocomotion/drake/pull/7911 here to make it script-style?
`from pydrake.all import *`?

Reply:
I'm not enough of a python pro to have a strong opinion.  but to make sure I understand, would this replace all of lines 4-7?